egov-indexer/hearing-booking-indexer.yml

```
@@ -47,7 +47,7 @@ ServiceMaps:
               - inJsonPath: $.hearing.auditDetails.lastModifiedTime
                 outJsonPath: $.Data.auditDetails.lastModifiedTime
             externalUriMapping:
-              - path: http://egov-workflow-v2.pucar-stg:8080/egov-workflow-v2/egov-wf/process/_search
+              - path: http://egov-workflow-v2.egov:8080/egov-workflow-v2/egov-wf/process/_search
                 queryParam: businessIds=$.hearing.hearingId,tenantId=$.hearing.tenantId,history=true
                 apiRequest: { "RequestInfo": { "apiId": "","ver": "1.0","ts": 1502890899493,"action": "asd","did": "4354648646","key": "xyz","msgId": "654654","requesterId": "61","authToken": "d9994555-7656-4a67-ab3a-a952a0d4dfc8","userInfo": { "id": 1,"uuid": "1fec8102-0e02-4d0a-b283-cd80d5dab067","type": "EMPLOYEE","tenantId": "pg","roles": [ { "name": "Employee","code": "EMPLOYEE","tenantId": "pg" } ] } } }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)